Makefile

```diff
@@ -49,10 +49,14 @@ ifeq ($(HAS_GO), yes)
 	CGO_CFLAGS ?= $(shell $(GO) env CGO_CFLAGS) $(CGO_EXTRA_CFLAGS)
 endif
 
-ifeq ($(OS), Windows_NT)
-	GOFLAGS := -v -buildmode=exe
-	EXECUTABLE ?= gitea.exe
-else ifeq ($(OS), Windows)
+ifeq ($(GOOS),windows)
+	IS_WINDOWS := yes
+else ifeq ($(findstring Windows,$(OS)),Windows)
```

Comment:
```suggestion
else ifeq ($(patsubst Windows%,,$(OS)),)
```

Test with this:

```make
OS := Windows_NT

all:
ifeq ($(patsubst Windows%,,$(OS)),)
	@echo "$(OS) starts with Windows"
else
	@echo "$(OS) doesn't start with Windows"
endif

Reply:
I agree, this is better. Updated.

Reply:
Actually this doesn't work when `OS` is empty

Reply:
A bit of a hack but seems to work with empty as well:

```
ifeq ($(patsubst Windows%,x,$(OS)),x)

Reply:
This is clearer:

```make
ifeq ($(patsubst Windows%,Windows,$(OS)),Windows)

Reply:
On the other hand it fails with spaces `OS = Windows NT`. Let's keep the current version.